Library/Formula/abcde.rb

```diff
@@ -22,6 +23,11 @@ class Abcde < Formula
   depends_on "vorbis-tools" => :optional
   depends_on "glyr" => :optional
 
+  patch do
+    url "http://abcde.einval.com/bugzilla/attachment.cgi?id=12"
+    sha256 "4620dd5ef7ab32b6511782da85831661ccee292fadff1f58acc3b4992486af62"
+  end
```

Comment:
Can you please submit this patch to the upstream developers of this project (if it hasn't been already) and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!